### DISCUSSION TITLE
tar_change repository not considered for change part

### DISCUSSION THREAD

**Discussion by SaJaToGu** [ropensci/tarchetypes#119] [General]
The _change object is not stored on the repository e.g. "aws".

**Comment by wlandau** [comment]
Right, that's because of the intent of the `_change` object. Its only should ideally run quickly and cheaply, and object should be small enough to comfortably keep locally. Uploading it to AWS  and tracking it there would cost API calls, which do not seem worth the trouble in this case.

**Comment by SaJaToGu** [comment]
I think the  advantage to store _change and maybe also the hole meat information is to be able to run the pipeline from anywhere.
If on machine is offline or you have no access to e.g. the RStudio Connect Server, you can still run pipelines from you local machine as fallback if you can reach the repository. 
Or even be able to debug problems locally in case there are problem on the RStudio Connect Server.
